### ISSUE TITLE
[COGNITION]: Charts need a longer text description of data or a table next to them for assistive tech

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#6027]
## Description

The PIRATE program page has two charts that show relative change and performance scores for three unique groups. This data is not described to screen readers, so some users will not be able to interpret the data about PIRATE. Screenshot below.

## Type of Issue
content

## Related Issue
* https://github.com/department-of-veterans-affairs/va.gov-team/issues/5987

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

**Definition of done**
* [ ] Screen reader users can parse the data by reading through a proper HTML table **OR**
* [ ] Screen reader users can listen to the data in a longer text description

## Environment

* MacOS Mojave
* Safari
* VoiceOver
* https://www.va.gov/pittsburgh-health-care/programs/pirate/

## WCAG or Vendor Guidance (optional)

* [Info and Relationships: Understanding SC 1.3.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html)

## Screenshots or Trace Logs

![Screen Shot 2020-02-19 at 4 53 37 PM](https://user-images.githubusercontent.com/934879/74884536-0926cb00-5339-11ea-85d2-92911f9dde10.png)

**Comment by davidconlon** [comment]
@1Copenut what are some options for this content.  The first seems pretty straight-forward, but the second might be more complex and potentially a content-owner training issue.

For example: The second image I would guess should have a "aria-describedby" referencing a table.  And then there is a question of whether or not the table is visible only to screen readers (and is that a good idea?)

My knee-jerk reaction here is this seems fairly straightforward but as we roll things out on the site could get more challenging with more intensive infographics.  

Would love some thought on this on how to potentially handle within the design system

**Comment by 1Copenut** [comment]
@davidconlon We're in new territory here, so we can write our playbook. 

I've talked with some friends in the healthcare space who have done similar graphic charts in the past. The common guidance has been to offer a pattern like:

* Meaningful heading
* 1-2 sentence explanation like the **Figure 1...** text in our example
* An image with a basic `alt` description
* A data table with a meaningful caption that could reiterate the Figure 1 text

The table should be available on the page for all users, with an emphasis on a semantic relationship between the heading, explainer, image, and table.

It's for sure an easier starting point than the second option. However, I think there's room for both, depending on the situation. Some data is easily explained in a table, like these images. There's only a few data points and we can easily add a column to calculate change over time if we desire.

For charts like time-series data over longer periods, we would probably need to lean on a `<figure> + <figcaption>` pattern to explain what the chart represents. If wait times are trending down over the last two years, with a big spike in the middle because of a policy change, that's relevant and wouldn't be easily called out in a table.

There's an education component to this, for sure. I think a first step could be to mock up a Codepen or something therein, with different charts and data displays (table, long form) as a learning and testing piece.
